julia/.travis.yml

```diff
@@ -0,0 +1,59 @@
+# Documentation: http://docs.travis-ci.com/user/languages/julia/
```

Comment:
I think it would be feasible to re-introduce Travis since you have been working with that package so far. I personally wouldn't really have time to maintain it though - would you be up to do it?

Reply:
Yeah. I can maintain it.
What I expect is that we still need travis build for the mirror, and it serves for auto-deploying the document.

Reply:
Travis is too slow. We should bring CI in to jenkins

Reply:
Auto-deploying what document? I think it would be best if we migrate all jobs to Jenkins - feel free to let me know if you got any problems with required dependencies.

Reply:
(see https://github.com/apache/incubator-mxnet/pull/8727#issuecomment-369365031)
The blueprint in my mind:
1. We have Jenkins build in main repo.
2. Set up mirror link with MXNet.jl
3. Just push to MXNet.jl and trigger Travis CI and Appveyor builds.
4. (I want to keep current doc setup:) Travis CI will push document to MXNet.jl's GitHub page.

If you really want to merge outputs of two document system (`Sphinx` and `Documenter.jl`),
I do not have any clear idea about detail of Sphinx.

Reply:
About the doc build: We can have to two separate doc jobs, that's not an issue. At the moment, this job is handled in the Deploy-stage - feel free to add another one for the Julia docs in case they use a different system.

The problem with your blueprint is that it does not cover bugs being introduced with PRs. it would be desirable to have Julia tests running as part of our PR stage. Where exactly do you see a problem in migrating from Travis and AppVeyor to Jenkins?